nft/emscripten/assemble.c

```diff
@@ -100,7 +100,7 @@ float EMSCRIPTEN_KEEPALIVE MIN(int x ,int y){
 }
 
 //------------AUX-FUNCTIONS-------------
-
+/*

```

Comment:
I had a linking error with those functions too, the only to solve was to include them in the file. But since you solved the linking error with the libjpeg, maybe it will not be a error anymore.